delta/sdk-testkit/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/testkit/RealmsBehaviors.scala

```diff
@@ -188,6 +191,40 @@ trait RealmsBehaviors {
       realms.deprecate(github, 3L).rejectedWith[RealmAlreadyDeprecated]
     }
 
+    val quote     = Pattern.quote("$")
+    val allEvents = List(
+      (github, RealmCreated.getClass.getSimpleName.replaceAll(quote, ""), Sequence(1L)),
```

Comment:
Seems that we are using this method to get the name of a class in many places.
Another variation was:
```scala
value.getClass.getSimpleName.split('$').head
```

In any case, shouldn't we put that somewhere in the SDK as some utils? (it is use in all the rejections for example).

However we can do that in a separate PR if you want.

Reply:
Yep, we can do it in another PR to avoid more noise